bin/dry-run.rb

```diff
@@ -85,6 +86,8 @@
 require "dependabot/python"
 require "dependabot/terraform"
 
+Dependabot.logger = Logger.new($stdout)
```

Comment:
That's already the default right?

Reply:
Set it to a nil logger by default here so existing users of dependabot-core don't get logging output without asking  for it: https://github.com/dependabot/dependabot-core/pull/2979/files#diff-ff1db9597b04517023f14d0ada4d0cb69112beb6b8ea697088cd15d3ce07d2fbR7